libs/seo/tsconfig.lib.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../out-tsc/lib",
+    "target": "es2015",
+    "declaration": true,
+    "inlineSources": true,
+    "types": [],
+    "lib": [
+      "dom",
+      "es2018"
+    ]
+  },
+  "angularCompilerOptions": {
```

Comment:
Can you compare these against a new ng12 package. We haven't reviewed these in a while, and it's worth it just to gloss over and compare.

Reply:
looking at the [schematic template](https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/library/files/tsconfig.lib.json.template), they look fairly similar.

Here's what `ng g library` outputs (for our current ng version):
```json
{
  "extends": "../tsconfig.json",
  "compilerOptions": {
    "outDir": "../out-tsc/lib",
    "target": "es2015",
    "declaration": true,
    "declarationMap": true,
    "inlineSources": true,
    "types": [],
    "lib": [
      "dom",
      "es2018"
    ]
  },
  "angularCompilerOptions": {
    "skipTemplateCodegen": true,
    "strictMetadataEmit": true,
    "enableResourceInlining": true
  },
  "exclude": [
    "src/test.ts",
    "**/*.spec.ts"
  ]
}
```